src/ray/raylet/node_manager.cc

```diff
@@ -445,7 +445,9 @@ NodeManager::NodeManager(instrumented_io_context &io_service,
     node_manager_server_.RegisterService(ray_syncer_service_);
   }
   node_manager_server_.Run();
-
+  // Set the node health status to be alive
```

Comment:
let's add inline comments in both client/server explain health check service will fail if the service name mismatches.